apis/v1alpha1/gateway_types.go

```diff
@@ -25,6 +25,8 @@ import (
 // +kubebuilder:resource:categories=gateway-api,shortName=gtw
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Class",type=string,JSONPath=`.spec.gatewayClassName`
+// +kubebuilder:printcolumn:name="Address",type=string,JSONPath=`.status.addresses[*].value`
+// +kubebuilder:printcolumn:name="Ready",type=string,JSONPath=`.status.conditions[?(@.type=="Ready")].status`
```

Comment:
Suggest removing from v1alpha1 since we haven't made any other v1alpha1 changes since v0.3.0

Reply:
done